django_tasks/backends/database/management/commands/db_worker.py

```diff
@@ -91,6 +91,9 @@ def start(self) -> None:
                         # "claim" the task, so it isn't run by another worker process
                         task_result.claim()
 
+                    else:
+                        time.sleep(self.interval)
```

Comment:
Issue: This holds the locking transaction open longer than necessary, which will affect throughput. This should probably be in the condition below, once the transaction has closed.

Reply:
Whoops, good point! Fixed.